roles/ceph-haproxy/tasks/pre_requisite.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: install haproxy
+  package:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - haproxy
+  when:
+    - ansible_os_family == 'RedHat'
+
+- name: "generate haproxy configuration file: haproxy.cfg"
+  action: config_template
```

Comment:
Please use default Ansible`template` instead.

Reply:
ok, I shall fix this.

Reply:
template module is a better idea because you could add file syntax validation via `validate: 'haproxy -f %s -c`
You should also add a handler to reload the config when config change. Currently any changes in the config won't be effective.

Reply:
@dsavineau thanks, I shall add validation and reloading.